test/components/form_test.exs

```diff
@@ -156,4 +156,28 @@ defmodule Surface.Components.FormTest do
 
     assert html =~ ~s(><input name="_method" type="hidden" value="put"/>)
   end
+
+  test "setting the class" do
+    html =
+      render_surface do
+        ~H"""
+        <Form for={{:user}} action="#" class="form">
+        </Form>
+        """
+      end
+
+    assert html =~ ~r/class="form"/
+  end
+
+  test "setting multiple classes" do
+    html =
+      render_surface do
+        ~H"""
+        <Form for={{:user}} action="#" class="form form-user">
```

Comment:
```suggestion
        <Form for={{:user}} action="#" class={{ "form", "form-user": true }}>
```

Just to make sure it accepts `:css_class` ;)

Reply:
I added it as a separate test.

In other form components we always test for class as single string `class="input"` and multiple strings `class="input primary"` (this form used to have a bug), but we don't test for `class={{ "input", "primary": true }}`, should I add it to all components?

Reply:
> should I add it to all components?

No need to do that now, however, we should consider a way to avoid repeating all those tests for each input. Currently,  42% of all tests are related to form/inputs components. Most of them are repeated. We can try to address this after `v0.4` is out.

Reply:
You are absolutely right! I've been thinking about this, but don't have a plan yet...